newroutes/netlifyToml.js

```diff
@@ -0,0 +1,36 @@
+const autoBind = require("auto-bind")
+const express = require("express")
+
+const { attachReadRouteHandlerWrapper } = require("@middleware/routeHandler")
+
+class NetlifyTomlRouter {
+  constructor({ netlifyTomlService }) {
+    this.netlifyTomlService = netlifyTomlService
+    // We need to bind all methods because we don't invoke them from the class directly
+    autoBind(this)
+  }
+
+  // Read netlify.toml file
+  async readNetlifyToml(req, res) {
```

Comment:
not a comment wrt this in particular but i realised that, in general, we have no idea what our functions throw. ie, all our router is showing is the happy path that the code takes and we rely upon the error handler middleware to transform and return errors.

my concern is that this causes devs to be unaware of possible errors (and how to handle them), instead deferring it to the frontend to format and display the error messages. some preconditions are also not clearly obvious - this requires the jwt to be authenticated (and we do so by having using hte auth middleware) but this isn't clear. 

i think the precondition section can be mitigated somewhat with proper ts typings but i'm not sure on hte error section. it'd be good to get your thoughts on this as well!

Reply:
also, i realised we have no proper integration tests HAHA

Reply:
As discussed offline, this will be tackled as part of a larger error handling cleanup in a future PR! Relevant issue: https://github.com/isomerpages/isomercms-backend/issues/352